backend/requirements.txt

```diff
@@ -8,3 +8,7 @@ flask-migrate
 flask-sqlalchemy
 flask-swagger-ui
 bcrypt
+marshmallow-sqlalchemy
```

Comment:
Serge suggested to specify the version of packages we use. I think you've used the newest one:
marshmallow-sqlalchemy==0.26.1
flask-marshmallow==0.14.0